lib/nuget-parser/cli/dotnet.ts

```diff
@@ -110,8 +110,11 @@ export async function publish(
   // Some projects can have <PublishSingleFile> turned on, that won't generate the self-container binary we need,
   // so we're disabling it during our scan.
   // See https://learn.microsoft.com/en-us/dotnet/core/deploying/single-file/overview?tabs=cli
+
+  // Some projects can have <TreatWarningsAsErrors> tuned on, that will throw errors on any warning, making the project impossible to scan.
+  // Or, they can have a list of warning codes in <WarningsAsErrors> that will do the same thing as above. So we're disabling them.
   args.push(
-    `--p:PublishDir=${tempDir};IsPublishable=true;PublishSingleFile=false`,
+    `--p:PublishDir=${tempDir};IsPublishable=true;PublishSingleFile=false;TreatWarningsAsErrors=false;WarningsAsErrors=`,
```

Comment:
should it be
`--p:PublishDir=${tempDir};IsPublishable=true;PublishSingleFile=false;TreatWarningsAsErrors=false;WarningsAsErrors=false?`

Reply:
No, WarningsAsErrors it's a list of warning codes, I just need to clear out that list
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/errors-warnings#warningsaserrors-and-warningsnotaserrors